fjord/analytics/static/css/dashboard.less

```diff
@@ -31,6 +31,7 @@
 }
 
 .block.feedback {
+  overflow: scroll;
```

Comment:
We don't want to do this here because it affects all the feedback blocks on the site including the front page dashboard.

It's better to add another css class with the `overflow:scroll` and add it to the specific div that's having difficulties.